### PR TITLE
SOC-1452 | Display deleted replies for moderators

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1452

### CODE REVIEW

File: front/scripts/main/models/DiscussionPostModel.ts [hunk 1]
```diff
@@ -24,7 +24,8 @@ App.DiscussionPostModel = App.DiscussionBaseModel.extend({
 						'sortKey': 'creation_date',
 						'limit': this.replyLimit,
 						'pivot': this.pivotId,
-						'page': this.page+1
+						'page': this.page + 1,
+						'viewableOnly': false
```
Comment (bkoval) [inline front/scripts/main/models/DiscussionPostModel.ts]: readOnly? :)

**Comment by bkoval** [comment]
LGTM :santa:

**Comment by rafalkalinski** [comment]
LGTM

**Comment by kvas-damian** [comment]
Merged to SOC-1451

**Pull request closed without merging**